inc/lang/en/lang.php

```diff
@@ -319,6 +319,8 @@
                                   You should either re-extract the files from the downloaded package or consult the complete
                                   <a href="http://dokuwiki.org/install">Dokuwiki installation instructions</a>';
 $lang['i_funcna']              = 'PHP function <code>%s</code> is not available. Maybe your hosting provider disabled it for some reason?';
+$lang['i_funcnmail']              = 'PHP function <code>%s</code> is not available. Maybe your hosting provider disabled it for some reason?'.
+                                 ' If it remains unavailable, install the <a href="http://dokuwiki.org/plugins/smtp">smtp plugin</a>.';
```

Comment:
Since this is a specific error message for the mail function, there is no need to pass `mail` as a parameter here. It would probably be a good idea to tell users what it means for them (that no mails will be sent).